02week/pigLatin.js

```diff
@@ -9,8 +9,39 @@ const rl = readline.createInterface({
 
 
 const pigLatin = (word)  => {
+  
+    const wordtrim = word.trim().toLowerCase();
```

Comment:
the logic is correct, but the function should return the translated word, instead of printing it out.